addons/web/static/src/views/kanban/kanban_examples_dialog.scss

```diff
@@ -1,45 +1,51 @@
 // container ghost card in modal
-.o_kanban_examples_dialog .modal-body {
-    display: flex;
-    padding: 0px;
-    background: map-get($theme-colors, 'light');
+.o_kanban_examples_dialog {
+    .modal-body {
+        display: flex;
+        padding: 0;
+        background: map-get($theme-colors, 'light');
+    }
 
-    // sidebar
-    .o_notebook_headers {
-        flex: 0 0 auto;
-        border-right: 1px solid map-get($grays, '300');
-        background: white;
+    .o_notebook {
+        --notebook-link-border-color: #{map-get($grays, '300')};
+        --notebook-link-border-color-active-accent: #{$o-brand-primary};
 
-        > ul > li {
-            margin: 0;
-            border-bottom: 1px solid map-get($grays, '300');
+        // sidebar
+        .nav-tabs {
+            border-bottom-color: transparent;
+        }
 
-            > a {
-                padding: 12px 15px;
-                border-left: 3px solid transparent;
-                &:not(.active) {
-                    color: $o-main-text-color;
-                }
-                &:focus {
+        .nav-item {
+            margin-bottom: 0;
+        }
+
+        .nav-link {
+            border-left-width: 3px;
```

Comment:
Consistency, if we decide that the width of the "active border" must `3px`, then we should do it globally and directly in the component SCSS.
Otherwise we'll get `1px` by default, `3px` here and whatever somewhere else 😉